devices/MAX1704x/Max17048.cs

```diff
@@ -0,0 +1,453 @@
+        protected override float Divider => 65536.0f;
+
+        /// <summary>
+        /// Gets or sets the reset voltage of a device.
+        /// </summary>
+        /// <exception cref="ArgumentOutOfRangeException">Thrown when the provided value is outside the valid range of 0 to 5.08.</exception>
+        public float ResetVoltage
+        {
+            get
+            {
+                var value = Read16(Registers.Max17048VresetId) >> 9;
+                return value * 0.04f;
+            }
+
+            set
+            {
+                if (value < 0 || value > 5.08)
+                {
+                    throw new ArgumentOutOfRangeException();
+                }
+
+                var thresh = (byte)(value / 0.04);
+                var vreset = Read16(Registers.Max17048VresetId);
+                vreset &= 0x01FF;
+                vreset |= (ushort)(thresh << 9);
+                Write16(vreset, Registers.Max17048VresetId);
+            }
+        }
+
+        /// <summary>
+        /// Gets the status of the property.
+        /// </summary>
+        /// <returns>The status as a byte value.</returns>
+        private byte Status
+        {
+            get
+            {
+                var statusReg = Read16(Registers.Max17048Status) >> 8;
+                return (byte)(statusReg & 0x7F);
+            }
+        }
+
+        /// <summary>
+        /// Gets the change rate of the property.
+        /// </summary>
+        /// <remarks>
+        /// This property returns the change rate of a certain value that is obtained from the MAX17048_CRATE register.
+        /// The value is then multiplied by 0.208 to convert it to the desired float value.
+        /// </remarks>
+        /// <value>
+        /// The change rate of the property.
+        /// </value>
+        public float ChangeRate
+        {
+            get
+            {
+                var changeRate = Read16(Registers.Max17048Crate);
+                return changeRate * 0.208f;
+            }
+        }
+
+        /// <summary>
+        /// Gets the ID of the property.
+        /// </summary>
+        /// <remarks>
+        /// This property returns the ID of the specified property. It retrieves the value from the
+        /// MAX17048_VRESET_ID register and returns the lower 8 bits as a byte.
+        /// </remarks>
+        /// <returns>The ID of the property as a byte.</returns>
+        public byte Id
+        {
+            get
+            {
+                var vresetID = Read16(Registers.Max17048VresetId);
+                return (byte)(vresetID & 0xFF);
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets maximum voltage threshold in volts for the VALRT interrupt.
+        /// </summary>
+        /// <remarks>
+        /// This property gets or sets the maximum voltage threshold for the VALRT interrupt.
+        /// The value is in volts, ranging from 0 to 5.1 volts.
+        /// </remarks>
+        public float VALRTMax
```

Comment:
here as well, as it's a Voltage, please use UnitsNet.
And second point, it's about the name, it's not the most speaking name :-) Any way to make it better? And also respect a bit better the naming convention?